### ISSUE TITLE
200Something went wrong... Code: 403 with qBittorrent WebUI

### ISSUE THREAD

**Issue by steakhutzeee** [linuxserver/Heimdall#547]
Hi :)

On my Odroid HC2 running OMV5 on top of Armbian i set up heimdall with this container https://hub.docker.com/r/linuxserver/heimdall

I'm also using qBittorrent with this container https://hub.docker.com/r/linuxserver/qbittorrent

I can reach the qBittorrent container just fine via Heimdall but when i try to put in my Web UI credentials in Heimdall to see the Enhanced App i receive the following pop up error:

"200Something went wrong... Code: 403"

Any idea why?

Thanks!

**Comment by bn326160** [comment]
Wanted to pitch in to mention I have the same issue on Docker on Ubuntu Server LTS 20.04.

However, when I added it anyway, it seems to be working?

**Comment by steakhutzeee** [comment]
Yes! It actually works, even with that error. That was not working when i opened the issue. Strange.

**Comment by jmu** [comment]
I googled and found the solution by reddit user u/joseamaria
Web UI Tab > Enable Cross-Site Request Forgery (CSRF) protection

https://www.reddit.com/r/docker/comments/gfkwwq/qbittorrent_webui_linking_problem/

**Comment by steakhutzeee** [comment]
> 
> 
> I googled and found the solution by reddit user u/joseamaria
> Web UI Tab > Enable Cross-Site Request Forgery (CSRF) protection
> 
> https://www.reddit.com/r/docker/comments/gfkwwq/qbittorrent_webui_linking_problem/

Hi,
the issue posted by the reddit user is different by mine.

Anyway i have that option disabled by long time and the issue is still present.

**Comment by steakhutzeee** [comment]
Actually with latest version of qbittorrent and heimdall the issue is present again, nothing happens when using qbittorrent credentials.

**Comment by steakhutzeee** [comment]
Hi,

the issue i mentioned on 29 Aug is still present as of today in November.

Something changed in the qbittorrent API and Heimdall has not yet been updated to reflect this change?

**Comment by ahmedelakkad** [comment]
r

> Actually with latest version of qbittorrent and heimdall the issue is present again, nothing happens when using qbittorrent credentials.

Same!
